reconcile/terraform_resources.py

```diff
@@ -238,9 +238,13 @@ def init_working_dirs(accounts, thread_pool_size,
     return ts, working_dirs
 
 
-def setup(print_only, thread_pool_size, internal, use_jump_host):
+def setup(print_only, thread_pool_size, internal, use_jump_host,
+          account_name):
     gqlapi = gql.get_api()
     accounts = queries.get_aws_accounts()
+    if account_name:
+        accounts = [n for n in accounts
```

Comment:
should we check that `accounts` is not empty?

Reply:
If it runs `--account-name` with nothing, get `Error: --account-name option requires an argument`.
If it runs with `--account-name ""`, it runs for all accounts.

Reply:
If no account matches, it only runs the oc part.

Reply:
Can we avoid it by checking?

Reply:
Sure, raise an error for account not found